ddl/ddl_worker.go

```diff
@@ -32,10 +34,57 @@ import (
 // RunWorker indicates if this TiDB server starts DDL worker and can run DDL job.
 var RunWorker = true
 
-// onDDLWorker is for async online schema changing, it will try to become the owner firstly,
+type workerType byte
+
+const (
+	// generalWorker is the worker who handles all DDL worker now.
+	// TODO: update the comments when we support the addIdxWorker.
+	generalWorker workerType = 0
+	addIdxWorker  workerType = 1
+)
+
+// worker is used for handling DDL jobs.
+// Now we have two kinds of workers, but we only use the generalWorker.
+// TODO: update the comments when we support the addIdxWorker.
+type worker struct {
```

Comment:
Could we add an ID for the worker? There will be more than one worker. So we need to distinguish them.